### PR TITLE
update lykke logo that is used in the header

### PR DESCRIPTION
**Pull request by nickhow83**
and update support for old nodejs on newer machines

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```
Comment (nickhow83) [inline .devcontainer/devcontainer.json:1]: This allows us to open the project in vscode, using a dev container.  Meaning we can install any version of Node inside the container, and not affect our local machines.

File: .devcontainer/dockerfile.dev [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  && nvm use default \
+  && node -v \
+  && npm -v
+
+# Update the $PATH to make your installed `node` and `npm` available!
+ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+
+RUN npm i -g yarn
+
+RUN sudo apt -y install python2
```
Comment (nickhow83) [inline .devcontainer/dockerfile.dev:29]: One of the libraries uses python2, so it needs installing

File: .devcontainer/dockerfile.dev [hunk 3]
```diff
@@ -0,0 +1,29 @@
+FROM mcr.microsoft.com/devcontainers/base:ubuntu
+
+SHELL ["/bin/bash", "--login", "-c"]
+
+RUN sudo apt update
+RUN sudo apt -y install curl
+
+# Set this to any desired version
+ENV NODE_VERSION 9.11.1
```
Comment (nickhow83) [inline .devcontainer/dockerfile.dev:9]: One of the packages has a maximum node major version of 9

File: src/Header/style.scss [hunk 4]
```diff
@@ -14,6 +14,7 @@
       margin: 13px 0 15px 40px;
       background: url('../img/logo.svg');
       background-repeat: no-repeat;
+      background-size: contain;
```
Comment (nickhow83) [inline src/Header/style.scss:17]: The image dimensions are different, so we can just container the image within it's parent container

File: tsconfig.json [hunk 5]
```diff
@@ -12,7 +12,7 @@
       "forceConsistentCasingInFileNames": true,
       "noImplicitReturns": true,
       "noImplicitThis": true,
-      "noImplicitAny": true,
+      "noImplicitAny": false,
```
Comment (nickhow83) [inline tsconfig.json:15]: I couldn't find some types, and this was breaking Storybook, so I turned it off.

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**